components/desktop/freerdp/Makefile

```diff
@@ -0,0 +1,111 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2019 Michal Nowak
+# Copyright 2020 Marco van Wieringen
+#
+
+BUILD_BITS= 64
+BUILD_STYLE= cmake
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		FreeRDP
+COMPONENT_VERSION=	2.2.0
+COMPONENT_COMMIT=	d2ba84a6885f57674098fe8e76c5f99d880e580d
+COMPONENT_SUMMARY=	Remote Desktop Viewer Client
+COMPONENT_PROJECT_URL=	https://www.freerdp.com
+COMPONENT_FMRI=		desktop/remote-desktop/freerdp
+COMPONENT_CLASSIFICATION=	Applications/Internet
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_COMMIT)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).zip
+COMPONENT_ARCHIVE_URL=	https://github.com/FreeRDP/FreeRDP/archive/$(COMPONENT_COMMIT).zip
+COMPONENT_ARCHIVE_HASH=	\
```

Comment:
I just checked: You don't need a COMPONENT_COMMIT anymore because there are releases already (haven't been there when Mno-hime worked on this):
https://github.com/FreeRDP/FreeRDP/releases/download/2.2.0/freerdp-2.2.0.tar.gz